### PR TITLE
Add new action T091 - rough diamonds

### PR DESCRIPTION
**Pull request by huwd**
relates to: https://trello.com/c/kYuHCUKy/190-t091-rough-diamonds-trade-fco
- Add: new action T091 - rough diamonds
- Testable Here: https://finder-frontend-pr-1622.herokuapp.com/get-ready-brexit-check/results?c%5B%5D=diamond&c%5B%5D=owns-operates-business-organisation

### CODE REVIEW

File: lib/brexit_checker/change_notes.yaml [hunk 1]
```diff
@@ -42,3 +42,7 @@ change_notes:
      action_id: T064
      note: "Link changed to provide hauliers with updated guidance."
      date: 2019-10-02
+   - uuid: "c23214f5-eee3-4528-88b1-c7d88fe59b7e"
+     type: addition
+     action_id: T091
+     date: 2019-10-03
```
Comment (benthorner) [inline lib/brexit_checker/change_notes.yaml]: We should update this to today's date.

**Comment by huwd** [comment]
amended

File: lib/brexit_checker/change_notes.yaml (hunk 1 above)
Comment (huwd) [reply lib/brexit_checker/change_notes.yaml]: thanks, amended

File: lib/brexit_checker/change_notes.yaml [hunk 2]
```diff
@@ -42,3 +42,7 @@ change_notes:
      action_id: T064
      note: "Link changed to provide hauliers with updated guidance."
      date: 2019-10-02
+   - uuid: "c23214f5-eee3-4528-88b1-c7d88fe59b7e"
+     type: addition
+     action_id: T091
+     date: 2019-10-07
```
Comment (huwd) [inline lib/brexit_checker/change_notes.yaml]: Changed this date in hopes we can get this up today

**Pull request merged**